farmdata2_modules/fd2_tabs/fd2_example/ex1.html

```diff
@@ -29,12 +30,14 @@
     components: {
         // Register the components that are being used.
         'fields-dropdown': fieldsDropdownComponent,
```

Comment:
The fields-dropdown component should not be registered since that component is no longer being used.